### PR TITLE
Fix/relay requests

### PR DESCRIPTION
**Pull request by carpawell**

### CODE REVIEW

File: pkg/services/object/put/distributed.go [hunk 1]
```diff
@@ -325,6 +325,7 @@ func (x placementIterator) iterateNodesForObject(obj oid.ID, f func(nodeDesc) er
 						return
 					}
 				}); err != nil {
+					wg.Done()
```
Comment (roman-khimov) [inline pkg/services/object/put/distributed.go:328]: A bit different (and incomplete) story.

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ attribute, which is used for container domain name in NNS contracts (#2954)
 - RPC client reconnection failures leading to complete SN failure (#2797)
 - `meta.DB.Open(readOnly)` moves metabase in RO mode (#3000)
 - Panic in event listener related to inability to switch RPC node (#2970)
+- Non-container nodes never checkes placement policy on PUT requests (#3014)
```
Comment (roman-khimov) [inline CHANGELOG.md:32]: checks

**Review by roman-khimov — APPROVED** [review]

File: pkg/services/object/put/distributed.go (hunk 1 above)
Comment (carpawell) [reply pkg/services/object/put/distributed.go:328]: agree, just did not want two PRs after the same debug session in fact

File: CHANGELOG.md [hunk 3]
```diff
@@ -29,6 +29,7 @@ attribute, which is used for container domain name in NNS contracts (#2954)
 - RPC client reconnection failures leading to complete SN failure (#2797)
 - `meta.DB.Open(readOnly)` moves metabase in RO mode (#3000)
 - Panic in event listener related to inability to switch RPC node (#2970)
+- Non-container nodes never checks placement policy on PUT requests (#3014)
```
[the remark this answers is not in the document]
Comment (carpawell) [reply CHANGELOG.md:32]: hm, how many devs are required to write this word right

**Pull request merged**